tasks/post_to_catalog/test/unit_tests/test_post_to_catalog.py

```diff
@@ -218,6 +218,7 @@ def test_create_catalog_records_happy_path(
             "etag": uuid.uuid4().__str__(),
         }
         granule = {
+            "providerId": uuid.uuid4().__str__(),
```

Comment:
Depending on what we decide with the input schema, this may need to be removed.

Reply:
removed